packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.ts

```diff
@@ -77,9 +79,24 @@ export function getGradableGraph(
     }
 
     if (state.type === "angle" && initialGraph.type === "angle") {
+        // We're going to reverse the coords for scoring if the angle is clockwise and
+        // we don't allow reflex angles. This is because the angle is largely defined
+        // by the order of the points in the coords array, and we want to maintain
+        // the same angle scoring with the legacy graph (which had a bug).
+        // (LEMS-2190): When we remove the legacy graph, move this logic to the scoring function.
+        const areClockwise = clockwise([
+            state.coords[0],
+            state.coords[2],
+            state.coords[1],
+        ]);
+        const shouldReverseCoords = areClockwise && !state.allowReflexAngles;
+        const coords: [Coord, Coord, Coord] = shouldReverseCoords
+            ? (state.coords.slice().reverse() as [Coord, Coord, Coord])
```

Comment:
I _think_ you can get away without the `as` if you use: 

```suggestion
            ? ([...state.coords].reverse()
```

Reply:
Sadly no luck here :( 

Reverse returns a generic array (Coord[]), and we're looking specifically for [Coord, Coord, Coord], so typescript becomes unhappy.

Reply:
Ah right!  It's `reverse()` that's part of the problem!